base/deprecated.jl

```diff
@@ -539,6 +539,63 @@ end
 
 export MathConst, @math_const
 
+# 11551
+function utf16_is_surrogate(chr::UInt16)
```

Comment:
This is a lot of exactly repeated code. Should really use a for loop and a macro, at least for all the single-arg cases.